metric/system/process/process.go

```diff
@@ -343,7 +341,7 @@ func (procStats *Stats) includeTopProcesses(processes []ProcState) []ProcState {
 
 // isWhitelistedEnvVar returns true if the given variable name is a match for
 // the whitelist. If the whitelist is empty it returns false.
-func (procStats Stats) isWhitelistedEnvVar(varName string) bool {
+func (procStats *Stats) isWhitelistedEnvVar(varName string) bool {
```

Comment:
I just made it consistent with the other receivers

Reply:
up to you, for me it's not about consistency, it's about semantics, if I have a pointer receiver in a function that means this function changes the state. Otherwise, you indicate that the function is read-only.